### DISCUSSION TITLE
叶绿体基因组组装不成环

### DISCUSSION THREAD

**Discussion by liuxing12311** [Kinggerm/GetOrganelle#344] [Q&A]
金老师您好，我在组装叶绿体时遇到了不成环的情况，按提供的建议优化以后，依旧不成环，想问下您的建议，07-3.log文件和07-3fastg图片，是增加了-w参数，增加了--reduce-reads-for-coverage=1000以及近缘参考序列，而07 -s.log文件与07 -s fastg图片是常规参数的，只加了一个近缘参考序列。谢谢老师！
[07-3 get_org.log.txt](https://github.com/user-attachments/files/16480568/07-3.get_org.log.txt)

![07-3-fastg图](https://github.com/user-attachments/assets/9f9b76e3-43cd-4787-b081-babd6a2a2a69)
[07 -s get_org.log.txt](https://github.com/user-attachments/files/16480583/07.-s.get_org.log.txt)
![07 -s fastg](https://github.com/user-attachments/assets/75f16ff3-943e-4022-a677-b75477782692)

**Comment by JianjunJin** [comment]
看图没有明显的高深度的dead end，所以大概率是organelle-sufficient，也就是说，extended assembly graph中应该包含了完整的叶绿体基因组（https://github.com/Kinggerm/GetOrganelle/wiki/FAQ#what-should-i-do-with-incomplete-resultbroken-assembly-graph），但不是很确定是否有很多repeats，这个图有比较多的mt的干扰。

可以在Bandage中load fastg和csv文件，然后配合深度手动清理掉一些mt以及和目标深度差异很大的contig，然后再把清理后的结果保存成gfa，用get_organelle_from_assembly.py导出fasta结果。

另外，也可以直接尝试GetOrganelle 1.8.0，那个的自动识别能力更强，目前还没有在conda上上线，但是可以通过如下方式安装：
```
 mamba create -n getorganelle2 python
 mamba activate getorganelle2
 mamba install -c bioconda blast
 pip install gekko
 git clone --depth 1 --branch intermediate_graph https://github.com/Kinggerm/GetOrganelle
 pip install ./GetOrganelle
```
